l1interop/l1sseclient.go

```diff
@@ -220,6 +221,18 @@ func (l *l1SseClient) Start(nConnectedl1s *atomic.Uint64) error {
 
 		nConnectedl1s.Dec()
 		log.Infow("lost connection to L1", "l1", l.l1Addr, "nL1sConnected", nConnectedl1s.Load())
+		reportConnectionStatus(nConnectedl1s)
+	}
+}
+
+func reportConnectionStatus(nConnectedl1s *atomic.Uint64) {
+	// Report the connection status to Filecoin Station
+	val := nConnectedl1s.Load()
+
+	if val > 0 {
+		fmt.Printf("INFO: Saturn Node is online and connected to %v peer(s)\n", val)
+	} else {
+		fmt.Print("ERROR: Saturn Node lost connection to the network\n")
```

Comment:
could be DRYed up, we have two pairs of 1. change state 2. general log line 3. saturn log line. wdyt?

Reply:
I am not sure if I understand your proposal correctly. Here are the different options:

1. New L1 connection is established - we report "online"
2. First or second L1 connection is lost - we report "online"
3. Last L1 connection is lost - we report "offline"

We have two state changes (1 and 2) where we want to report the same message. That's why I created this helper function to avoid duplication. Also to give this piece of code a descriptive name pointing to the Station.

Reply:
Sorry for not being clearer, I was wondering if we should include the updating of the value and the logging of the l2 log line in this function, so that we have one piece of logic for updating the value, which also takes care of logging its new state.